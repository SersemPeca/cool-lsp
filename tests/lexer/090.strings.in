"new line in string here:
not_same_string"
