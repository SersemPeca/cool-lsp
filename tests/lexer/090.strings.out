#1 ERROR: String contains unescaped new line
#2 OBJECTID not_same_string
#2 ERROR: Unterminated string at EOF
